sklearn/kernel_approximation.py

```diff
@@ -38,11 +38,10 @@ class RBFSampler(TransformerMixin, BaseEstimator):
         Number of Monte Carlo samples per original feature.
         Equals the dimensionality of the computed feature space.
 
-    random_state : int, RandomState instance or None, optional (default=None)
-        If int, random_state is the seed used by the random number generator;
-        If RandomState instance, random_state is the random number generator;
-        If None, the random number generator is the RandomState instance used
-        by `np.random`.
+    random_state : int, RandomState instance, default=None
+        Determines random generator of number.
```

Comment:
Again, this could provide more helpful detail.

Reply:
@jnothman So I scanned the random kitchen article and if I change that sentence to this "Determines randomized take of features." , is correct? Or is more detail needed? And do you think there is no need to write "if None then it is np.random? I find this information important.